docs/howto/serialize_poro.md

```diff
@@ -29,4 +29,10 @@ class MyModel < ActiveModelSerializers::Model
 end
 ```
 
-The default serializer would be `MyModelSerializer`.
\ No newline at end of file
+The default serializer would be `MyModelSerializer`:
+```ruby
+# my_model_serializer.rb
+class MyModelSerializer < ActiveModel::Serializer
+  attributes :id, :name, :level
+end
```

Comment:
Would you mind commenting here on how this change improves the documentation? There's definitely room for improvement, in particular, I don't think the word `default` is helpful.  It's more like the automatic lookup would resolve to a serializer with that name.  Besides that, how do you see adding an example serializer helpful?  I'd rather have a link to the serializers docs for more info.